gulpfile.js

```diff
@@ -6,51 +6,38 @@
   https://opensource.org/licenses/MIT.
 */
 
-/* eslint-disable no-console, valid-jsdoc */
-
+const fse = require('fs-extra');
+const glob = require('glob');
```

Comment:
Could we use `globby` here? My understanding is it's faster than `glob` and it also supports async/await.

Reply:
Sure, but it's going to be `globby.sync()` anyway, since it's a top-level call in our `gulpfile.js`, and I don't think that could be `await`-ed.